backend/src/appointment/controller/mailer.py

```diff
@@ -364,13 +384,17 @@ def html(self):
 
 class SupportRequestMail(Mailer):
     def __init__(self, requestee_name, requestee_email, topic, details, *args, **kwargs):
-        """init Mailer with support specific defaults"""
+        """Init Mailer with support specific defaults
+           To: Support
+           Reply-To: Requestee
+        """
         self.requestee_name = requestee_name
         self.requestee_email = requestee_email
         self.topic = topic
         self.details = details
         default_kwargs = {'subject': l10n('support-mail-subject', {'topic': topic})}
         super(SupportRequestMail, self).__init__(os.getenv('SUPPORT_EMAIL', 'help@tb.net'), *args, **default_kwargs, **kwargs)
+        self.reply_to = requestee_email
```

Comment:
This changed line is off-topic, sorry. But I thought it would be very convenient, to be able to just reply to a support request. Let me know if this is not wanted.